packages/@aws-cdk/aws-codepipeline-actions/lib/s3/source-action.ts

```diff
@@ -72,7 +72,17 @@ export interface S3SourceActionProps extends codepipeline.CommonAwsActionProps {
   readonly trigger?: S3Trigger;
 
   /**
-   * The Amazon S3 bucket that stores the source code
+   * The Amazon S3 bucket that stores the source code.
+   *
+   * If you import an encrypted bucket in your stack, please specify
+   * the encryption key at import time, for example:
+   *
+   * ```
+   * const bucket = s3.Bucket.fromBucketAttributes(this, 'Bucket', {
+   *   BucketName: 'MyBucket',
+   *    encryptionKey: kms.Key.fromKeyArn(stack, 'MyKey', 'arn:aws:kms:...'),
+   * });
```

Comment:
Let's not put code in the docs (remember that CDK is a multi-language environment).

Just mention the `Bucket.fromBucketAttributes()` method here, and that you should provide the Key there.

Or move these docs to the ReadMe file (those are automatically translated to other languages).